addons/azure-cloud-node-manager/cloud-node-manager.yaml

```diff
@@ -17,9 +17,6 @@
 {{ if .Cluster.Features.Has "externalCloudProvider" }}
 {{ if eq .Cluster.CloudProviderName "azure" }}
 {{ $version := "UNSUPPORTED" }}
-{{ if eq .Cluster.MajorMinorVersion "1.23" }}
-{{ $version = "v1.23.30" }}
-{{ end }}
 {{ if eq .Cluster.MajorMinorVersion "1.24" }}
 {{ $version = "v1.24.18" }}
 {{ end }}
```

Comment:
Can't we remove this as well? Since users need to be on k8s 1.25 or higher to upgrade to the new KKP minor.

Reply:
No. This PR removes support for _new_ clusters, but in general KKP 2.24 will still support upgrading from 1.24 to 1.25 and os needs to be able to reconcile existing 1.24 clusters.

Reply:
Okay